manifests/t/Timthreetwelve/WUView/0.5.11/Timthreetwelve.WUView.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with Komac v1.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Timthreetwelve.WUView
+PackageVersion: 0.5.11
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+UpgradeBehavior: install
+ReleaseDate: 2023-06-17
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
`x86` validations are performed on a 32-bit OS, so I'm guessing this is a 64-bit package.

---

You are right. I used Komac to create this PR, but it only checked the architecture of the installer file, which is 32-bit, and assumed the same for the installed application. I have implemented your suggestion, thanks for that.